app/models/beer.rb

```diff
@@ -1,11 +1,16 @@
 class Beer < ActiveRecord::Base
   include Authorization
+
+  mount_uploader :image, BeerUploader
+
   has_many :reviews, dependent: :destroy
   belongs_to :style
+  belongs_to :brewery
   belongs_to :user
 
-  validates :name, presence: true, uniqueness: true
+  validates :name, presence: true, uniqueness: { scope: :brewery }
```

Comment:
Took out the uniqueness constraint on beer name, but added a new constraint so no brewery can have the two beers with the same name.